xdn-nx-example/README.md

```diff
@@ -0,0 +1,95 @@
+# NxNext
+
+<b>XDN guides:</b>
+
+- [XDN](XDN.md)
+- [XDN: creating NX app connector guide](XDN.md#xdn-creating-nx-app-connector-guide)
+
+This project was generated using [Nx](https://nx.dev).
+
+<p align="center"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="450"></p>
+
+🔎 **Nx is a set of Extensible Dev Tools for Monorepos.**
+
+## Adding capabilities to your workspace
+
+Nx supports many plugins which add capabilities for developing different types of applications and different tools.
+
+These capabilities include generating applications, libraries, etc as well as the devtools to test, and build projects as well.
+
+Below are our core plugins:
+
+- [React](https://reactjs.org)
+  - `npm install --save-dev @nrwl/react`
+- Web (no framework frontends)
+  - `npm install --save-dev @nrwl/web`
+- [Angular](https://angular.io)
+  - `npm install --save-dev @nrwl/angular`
+- [Nest](https://nestjs.com)
+  - `npm install --save-dev @nrwl/nest`
+- [Express](https://expressjs.com)
+  - `npm install --save-dev @nrwl/express`
+- [Node](https://nodejs.org)
+  - `npm install --save-dev @nrwl/node`
+
+There are also many [community plugins](https://nx.dev/nx-community) you could add.
+
+## Generate an application
+
+Run `nx g @nrwl/react:app my-app` to generate an application.
```

Comment:
Which command installs `nx` globally to be used? It looks like the above commands only installs the module for an existing project.

Reply:
It's NX guide that is there by default after I generated a sample NX project